js-miniapp-sdk/src/miniapp.ts

```diff
@@ -35,6 +36,12 @@ interface MiniAppFeatures {
    * @returns The Promise of share info action state from injected side.
    */
   shareInfo(info: ShareInfoType): Promise<string>;
+
+  /**
+   * @param screenAction The action that miniapp wants to request on device.
+   * @returns The Promise of screen action state from injected side.
```

Comment:
So there's the possibility that the orientation wasn't changed to the one requested? Does this only happen if the Host App doesn't support changing the orientation?

I think we should add some description text above explaining that the orientation isn't guaranteed to have been changed.

Reply:
On Android we can force the swap change even though the user or hostapp has lock the orientation but I'm not sure for every devices. On iOS so far we know that it strictly depends on hostapp.
=> I will add the description in function and document about miniapp should not rely on sdk.

Reply:
Or maybe you can also two statements for iOS and Android separately